ITISFoundation/osparc-ops-environments
# Remove MAIL service

## What do changes do
These changes remove MAIL services from the repository. Since we no longer rely on self-hosted MAIL service, we don't need it anymore. Furthermore, it causes errors while deploying locally and causes certificates generations (`./certificates`) to fail due to missing mail-related variable

## How to test
Deploy OPS stack locally and ensure that MAIL services is not deployed and all services are up and running
